shotover-proxy/tests/kafka_int_tests/mod.rs

```diff
@@ -656,7 +687,7 @@ fn multi_shotover_events(driver: KafkaDriver) -> Vec<EventMatcher> {
             .with_level(Level::Warn)
             .with_target("shotover::transforms::kafka::sink_cluster")
             .with_message(
-                r#"no known coordinator for GroupId("some_group"), routing message to a random node so that a NOT_COORDINATOR or similar error is returned to the client"#,
+                r#"no known coordinator for GroupId("some_group"), routing message to a random broker so that a NOT_COORDINATOR or similar error is returned to the client"#,
```

Comment:
Fixes an unrelated intermittent failure that slip in through: https://github.com/shotover/shotover-proxy/pull/1750/files